forecasters/animalia/R/production_forecaster.R

```diff
@@ -148,6 +155,12 @@ production_forecaster <- function(df_list,
   df_list <- transformer(df_list, transform, inv_trans)
   df_wide <- covidcast::aggregate_signals(df_list, dt = dt, format = "wide")
   
+  # keep just the geo_values we want, then featurize
```

Comment:
I think this is actually in the wrong spot. Currently, it'll have the same issue as before: rather than selecting highest total cases, it'll select highest case rate. We should move this block above `normalize_by_population()`. Then it will take in the df_list rather than the wide df. So you'll need to update the selector below.

Reply:
Good catch. I've amended the selector function so that it takes in `df_list` instead of `df_wide` (because of this, it no longer needs the data source and signal arguments), and have amended the tests accordingly.

I have also changed production forecaster so that filtering by geo_value happens before any transformation or featurizing. Since all the transformation and featurizing we do is by geo_value, it is OK to remove geo_values we don't want before that step.